nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -75,6 +76,10 @@ def _get_minmax_quantization_param_grid() -> ParamGrid:
             RangeEstimatorParameters(min=min_v, max=max_v)
             for min_v, max_v in itertools.product(min_param_values, max_param_values)
         ],
+        "advanced_parameters:propagation_strategy": [
```

Comment:
Is it covered by existing tests?

Reply:
There is a test in `tests/common/hyperparameter_tuner/test_hyperparameter_tuner.py` which tests the inner function of the hyperparameter tuner, but I could not find any test that can be extended by this specific param.
So, in my assumption, if the algorithm works correctly, it should work for this new param if it is specified correctly. I specified this param by analog which is` advanced_parameters:activations_range_estimator_params` and `advanced_parameters:weights_range_estimator_params`.

Reply:
discussed with @andrey-churkin and agreed that it's enough